android-core/src/main/java/com/mparticle/internal/ConfigManager.java

```diff
@@ -566,6 +588,16 @@ public void setLogUnhandledExceptions(boolean log) {
         sPreferences.edit().putBoolean(Constants.PrefKeys.REPORT_UNCAUGHT_EXCEPTIONS, log).apply();
     }
 
+    public UploadSettings getUploadSettings() {
+        String apiKey = getApiKey();
+        String secret = getApiSecret();
+        if (apiKey == null || secret == null) {
+            return null;
+        }
+
+        return new UploadSettings(apiKey, secret, getNetworkOptions(), getActiveModuleIds(), getSupportedKitString());
+    }
+
```

Comment:
Actually I think this new line is supposed to be here because otherwise there would be no empty line between the `}` and the `public String getApiKey() {`